src/components/network/network-table.js

```diff
@@ -913,27 +942,33 @@ const NetworkTable = (props) => {
                                 classes.inlineEditionCell
                             )}
                             equipment={rowData}
-                            defaultValue={text}
-                            setcolerror={(k) => setColumnInError(k)}
-                            resetcolerror={(k) => resetColumnInError(k)}
-                            datakey={columnDefinition.dataKey}
+                            defaultValue={currentValue}
+                            setColumnError={(k) => setColumnInError(k)}
+                            resetColumnError={(k) => resetColumnInError(k)}
+                            forceColumnUpdate={forceLineUpdate}
```

Comment:
it will be great if you rename forceColumnUpdate={forceLineUpdate}  to this forceLineUpdate={forceLineUpdate}